gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # androidSdkVersions
-compileSdkVersion = "32"
+compileSdkVersion = "33"
```

Comment:
Is this the right PR to introduce these changes? 🤔 There seems to be a lot of version updates in here 😬

Reply:
The new deps are

androidx-compose-material3-windowSizeClass: To enable not stretching the image super wide when on a wide screen or on landscape mode.

androidx-customview-poolingcontainer: A bug which disables compose previews in feature module requires this dependency.

accompanist-placeholder: A lib with a simple compose Modifier to get a nice loading effect behind the image while it's still loading.

The rest of the changes are just sorting dependencies by name which given enough time just get mixed up a bit.

The 33 compileSdk is needed for the windowSizeClass one, but it's just a bump on the compile sdk, not the targetsdk, so that's a safe thing to do without changing the app's behavior. Plus we get new lint hints and stuff like that which we will need to be aware of for when we eventually target sdk 33. https://medium.com/androiddevelopers/picking-your-compilesdkversion-minsdkversion-targetsdkversion-a098a0341ebd